e2e-tests/cypress/integration/appellant-submission-decision-letter.feature

```diff
@@ -7,33 +6,37 @@ Feature: Appellant submission - decision letter
   * not exceed a size limit.
   The latest successfully uploaded decision letter file replaces any previously uploaded file.
 
+  Scenario: Prospective applicant do not upload a decision letter file
+    When user does not submit a decision letter file
+    Then user can see that no decision letter file is submitted
+
   Scenario Outline: Prospective appellant submits valid decision letter file
     When user submits a decision letter file <filename>
-    Then user can see that the decision letter file <filename> "is" submitted
+    Then The application file <filename> is submitted and user can proceed
     Examples:
-      | filename                         |
-      | "upload-file-valid.doc"          |
-      | "upload-file-valid.docx"         |
-      | "upload-file-valid.pdf"          |
-      | "upload-file-valid.tif"          |
-      | "upload-file-valid.tiff"         |
-      | "upload-file-valid.jpg"          |
-      | "upload-file-valid.jpeg"         |
-      | "upload-file-valid.png"          |
-      | "upload-file-valid-max-size.png" |
-
-  Scenario: Prospective appellant submits valid decision letter file to replace a previous file
-    Given user has previously submitted a decision letter file "upload-file-valid.pdf"
-    When user submits a decision letter file "upload-file-valid.doc"
-    Then user can see that the decision letter file "upload-file-valid.doc" "is" submitted
+      | filename                              |
+      | "appeal-statement-valid.doc"          |
```

Comment:
This will probably not affect performance so it is not a blocking change request but the generic upload-file-valid.doc type file names were introduced to avoid having to either create lots of very similar sample data files or to re-uses inappropriately named ones.